images/publish_fluentbit_image.sh

```diff
@@ -13,25 +16,46 @@ echo "*** WARNING! ***"
 echo ""
 read -p "Press enter to continue."
 
+
+echo ""
+echo "*** Logging in to ECR Private ***"
+echo ""
+AWS_PROFILE=platform-developer aws ecr get-login --no-include-email | bash
+
+echo ""
+echo "*** Logging in to ECR Public ***"
+echo ""
+
+# We have to log in with us-east-1 because that's where ECR Public lives.
+# See https://docs.aws.amazon.com/AmazonECR/latest/public/getting-started-cli.html#cli-authenticate-registry
+AWS_PROFILE=platform-developer aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws
+
+
 echo ""
-echo "*** Publishing fluentbit ***"
+echo "*** Building fluentbit image ***"
 echo ""
 
-"$ROOT"/scripts/docker_run.py \
-      --dind --root -- \
-      wellcome/image_builder:23 \
-            --project=fluentbit \
-            --file=images/fluentbit/Dockerfile
```

Comment:
Trying to find where these Docker images are defined and what they do was a bit of a faff. Given how relatively simple the shell commands are, I opted to redo the scripts rather than shell out to some mystery images.

Reply:
We almost never touch those images, and they (i think) are in the very unloved dockerfiles repo that i'd like to get rid of anyway.

Reply:
They’re in the *history* of that repo, not even the latest commit. 😭